index.rst

```diff
@@ -38,6 +38,11 @@ topics.
    dist_tables/extensions.rst
 
 .. toctree::
+   :caption: Migration
```

Comment:
I don't see this in the index in the temporary deployment btw. Are there other changes needed in the index in order to make it visible? https://citus-doc-preview-pr-168.herokuapp.com/